lib/RTCSession.js

```diff
@@ -17,7 +17,9 @@ const RTCSession_ReferNotifier = require('./RTCSession/ReferNotifier');
 const RTCSession_ReferSubscriber = require('./RTCSession/ReferSubscriber');
 const URI = require('./URI');
 
-const logger = new Logger('RTCSession');
+const { registerGlobals } = require('react-native-webrtc');
```

Comment:
You introduced Dependencies without an package and I don't think this project should depend on react-native-webrtc that does not make sense.

Reply:
Correct.